rustls/src/quic.rs

```diff
@@ -238,6 +334,176 @@ pub enum Version {
 mod test {
     use super::*;
 
+    #[test]
```

Comment:
Tests of the public API should be integration tests (under `tests/`), not unit tests, whenever practical.

Reply:
Moved them.